packages/api/src/external/commonwell/__tests__/organization.ts

```diff
@@ -0,0 +1,34 @@
+import { Organization as CWOrganization } from "@metriport/commonwell-sdk";
+import { Config } from "../../../shared/config";
+import { capture } from "../../../shared/notifications";
+import { OID_PREFIX } from "../../../shared/oid";
+import { Util } from "../../../shared/util";
+import { makeCommonWellAPI, metriportQueryMeta } from "../api";
+
+const commonWell = makeCommonWellAPI(Config.getCWMemberOrgName(), Config.getCWMemberOID());
+
+/**
+ * For E2E testing locally and staging.
+ */
+export const getOne = async (orgOid: string): Promise<CWOrganization | undefined> => {
```

Comment:
No i still have this for the get org stuff just to validate we can query an org and update it in cw

Reply:
Oh, I see. I used VSCode's "Find all references" on this function and it didn't return anything; it's because of the way the `index.ts` file is exporting it. If we move to the format below, VSCode can find references:

```ts
export * as organization from "./organization";
```

Just need to change the places that reference it to import with `* as cwCommands` instead of just `cwCommands`.